lighthouse-cli/test/fixtures/byte-efficiency/tester.html

```diff
@@ -55,6 +55,10 @@
   <!-- PASS(responsive): image is used at full size -->
   <img src="lighthouse-unoptimized.jpg">
 
+  <!-- FAIL(optimized): image is not optimized -->
+  <!-- PASS(responsive): image is used at full size -->
+  <img src="http://localhost:10503/byte-efficiency/lighthouse-unoptimized.jpg">
```

Comment:
looks like this won't work because it depends on the order of the network records coming in (it's slot 0 on travis for whatever reason). This might need to wait until we have function-based comparisons in smokehouse